docs/Walkthrough.md

```diff
@@ -73,3 +78,11 @@ For more information, check [Payment Requests](PaymentRequests.md)
 ## My Settings
 
 My Settings are user settings. Modify your e-mail address, password or set up two-factor authentication. The U2F feature allows users to further enhance the security of their accounts by physically confirming authentication with a U2F compatible device.
+
+## Notifications
+
+Notifications inform the user that an event has accured on the BTCPay Server instance.
+The event that is notified can for example be a received or failed payment, an overpaid or underpaid invoice.
+
+By clicking on the icon you can access the Notifications page, where you can view past Notifications and optionally delete them.
+For the time being, it is not possible to select Notification preferences.
```

Comment:
>  For the time being, it is not possible to select Notification preferences. 

Irrelevant to the user imo and will cause maintenance in the future, I suggest we remove that sentence.